invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/oaipmh/details/LinksTable.js

```diff
@@ -5,32 +5,136 @@
 // under the terms of the MIT License; see LICENSE file for more details.
 
 import PropTypes from "prop-types";
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { i18next } from "@translations/invenio_administration/i18next";
 import CopyButton from "./CopyButton";
-import { Table, Header } from "semantic-ui-react";
+import { Table, Header, Dropdown, Grid } from "semantic-ui-react";
+import { http } from "@js/invenio_administration/src/api/config.js";
+import { capitalize } from "lodash";
+
+/** Map of known formats and their name. */
+const knownFormats = {
```

Comment:
Used to have readable names for currently known OAI formats. 

If a format is added later, it won't be "known". To support it nonetheless, this component will display the unknown format in the dropdown.